src/main/java/ch/njol/skript/doc/HTMLGenerator.java

```diff
@@ -455,7 +455,10 @@ private String generateEvent(String descTemp, SkriptEventInfo<?> info) {
 		
 		assert desc != null;
 		desc = handleIf(desc, "${if events}", false);
```

Comment:
Does this need to be handled as well?

Reply:
Indeed, all these should be handled but I didn't do them as I saw none of them are used unless I am wrong, but to do it the propper way they all should be handled, so I will edit them all for sure :+1:

> Thanks for this PR :D

You're welcome :)